roles/edpm_ovn/tasks/configure.yml

```diff
@@ -62,20 +62,32 @@
 
 - name: Configure OVS external_ids
   ansible.builtin.shell: >
-    ovs-vsctl set open . {% for key, value in edpm_ovn_ovs_external_ids.items() %} external_ids:{{ key }}={{ value }} {% endfor %}
+    ovs-vsctl set open . {% for key, value in edpm_ovn_ovs_external_ids.items() -%} external_ids:{{ key }}={{ value }} {% endfor %}
+  register: ovs_external_ids
+  changed_when: ovs_external_ids.rc == 0
+  failed_when: ovs_external_ids.rc != 0
 
 - name: Configure OVS other_config
   ansible.builtin.shell: >
     ovs-vsctl set open . {% for key, value in edpm_ovn_ovs_other_config.items() %} other_config:{{ key }}={{ value }} {% endfor %}
+  register: ovs_other_config
+  changed_when: ovs_other_config.rc == 0
+  failed_when: ovs_other_config.rc != 0
 
 - name: Add OVS Manager
   block:
     - name: Check if OVS Manager already exists
       ansible.builtin.shell: >
+        set -o pipefail
         ovs-vsctl show | grep -q "Manager"
       register: ovs_manager_configured
-      ignore_errors: yes
+      changed_when: true
+      failed_when: false
+
     - name: Add OVS Manager if not exists
       ansible.builtin.shell: >
         ovs-vsctl --timeout=5 --id=@manager -- create Manager target=\"ptcp:6640:127.0.0.1\" -- add Open_vSwitch . manager_options @manager
       when: ovs_manager_configured.rc == 1
+      register: ovs_manager_if_not_exists
+      changed_when: ovs_manager_if_not_exists.rc == 0
+      failed_when: ovs_manager_if_not_exists.rc != 0
```

Comment:
Adoption CI started to fail after this change
https://github.com/openstack-k8s-operators/data-plane-adoption/pull/178#issuecomment-1758200099

Reply:
@fao89 https://github.com/openstack-k8s-operators/edpm-ansible/pull/440 fixes that issue. So once it's available in the jobs, issue shouldn't be seen